src/displayBackend/wayland/Connector.hpp

```diff
@@ -62,7 +62,7 @@ class Connector : public DisplayItf::Connector
 	 * Checks if the connector is initialized
 	 * @return <i>true</i> if initialized
 	 */
-	bool isInitialized() const override { return mInitialized; }
+	bool isInitialized() const override { return static_cast<bool>(mSurface); }
```

Comment:
Looks ambiguous IMO

Reply:
Why?

Reply:
Certainly not consistent with what comment above says.

Reply:
If connector is initialized means surface is created and mSurface keeps pointer on it.
mSurface is shared_ptr. Cast it to bool checks if shared_ptr keeps pointer to the object e.g.
if connector is initialized. Will mSurface != nullptr be less ambiguous in this place?

Reply:
I understand these bits. From my point of view mSurface != nullptr is more cleaner, but perhaps static_cast<bool>(mSurface) is a common way on C++, I don't really know.
But my question was in comment. Text says "Checks if the connector is initialized and return true if initialized", but I don't see exactly such actions being performed in the function. I understood what was going on, after you had clarified me about it.

Reply:
Good. I will merge as is.

Reply:
Without clarifying comment in code?

Reply:
I suppose mSurface != nullptr is appropriate way for this situation.

Reply:
np. Done.